tests/bench.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+from tank_vendor.shotgun_api3 import Shotgun
+import time
+
+sg = Shotgun("https://hubertp-studio.shotgunstudio.com", login="automation.admin", password="Admin.12345")
```

Comment:
looks like we either want to delete this script (keep in mind this is a public repo!) or we need to add a bunch of information - what is this script for, how do we run it, add std headers, docstring etc.